ruleset/sca/windows/cis_win10_enterprise.yml

```
@@ -942,30 +986,34 @@ checks:
     remediation: "To establish the recommended configuration via GP, set the following UI path to Enabled: Computer Configuration\\Policies\\Windows Settings\\Security Settings\\Local Policies\\Security Options\\Network security: Do not store LAN Manager hash value on next password change."
     compliance:
       - cis: ["2.3.11.5"]
-      - cis_csc: ["16.14"]
+      - cis_csc: ["3.11"]
       - pci_dss: ["8.2.1"]
       - tsc: ["CC6.1"]
     references:
-      - 'CCE-36326-7'
-    condition: all
+      - "CCE-36326-7"
+    condition: any
     rules:
+      - 'r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa'
+      - 'r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa -> NoLMHash'
       - 'r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa -> NoLMHash -> 1'
 
   # 2.3.11.6 Ensure 'Network security: Force logoff when logon hours expire' is set to 'Enabled'
   - id: 15059
     title: "Ensure 'Network security: Force logoff when logon hours expire' is set to 'Enabled'"
-    description: "This policy setting determines whether to disconnect users who are connected to the local computer outside their user account's valid logon hours. This setting affects the Server Message Block (SMB) component. If you enable this policy setting you should also enable Microsoft network server: Disconnect clients when logon hours expire (Rule 2.3.9.4). The recommended state for this setting is: Enabled. Note: This recommendation is unscored because there is not a documented registry value that corresponds to it. We still strongly encourage that it be configured as Enabled, to ensure that logon hours (when configured) are properly enforced."
+    description: "This policy setting determines whether to disconnect users who are connected to the local computer outside their user account's valid logon hours. This setting affects the Server Message Block (SMB) component. If you enable this policy setting you should also enable Microsoft network server: Disconnect clients when logon hours expire (Rule 2.3.9.4). The recommended state for this setting is: Enabled. Note: This recommendation is unscored because there is not a documented registry value that corresponds to it."
     rationale: "If this setting is disabled, a user could remain connected to the computer outside of their allotted logon hours."
     remediation: "To establish the recommended configuration via GP, set the following UI path to Enabled. Computer Configuration\\Policies\\Windows Settings\\Security Settings\\Local Policies\\Security Options\\Network security: Force logoff when logon hours expire."
     compliance:
       - cis: ["2.3.11.6"]
-      - cis_csc: ["16"]
+      - cis_csc: ["5.6"]
       - pci_dss: ["7.1"]
       - tsc: ["CC6.4"]
     references:
-      - 'CCE-36270-7'
-    condition: all
+      - "CCE-36270-7"
+    condition: any
     rules:
+      - 'r:HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\LanManServer\Parameters'
```

Using condition 'any' probably means this should be:
```suggestion
      - 'not r:HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\LanManServer\Parameters'
```

---

Done 18dbade